From ab57dad70829d1aa7fc1afb2e355a1a337dd12a9 Mon Sep 17 00:00:00 2001
From: Jerome Audu <jau@free.fr>
Date: Mon, 30 Oct 2023 10:17:46 +0100
Subject: [PATCH] dts: reserve memory for uboot bootsplash

U-boot configuration
BR2_PACKAGE_DISPLAY_SPLASH_MEM_BASE="0x88000000"
BR2_PACKAGE_DISPLAY_SPLASH_MEM_SIZE="90177536
---
 arch/arm/boot/dts/st/stih407-family.dtsi | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/arch/arm/boot/dts/st/stih407-family.dtsi b/arch/arm/boot/dts/st/stih407-family.dtsi
index e72e274b2..8f4d8cd20 100644
--- a/arch/arm/boot/dts/st/stih407-family.dtsi
+++ b/arch/arm/boot/dts/st/stih407-family.dtsi
@@ -28,6 +28,11 @@ delta_reserved: rproc@44000000 {
 			reg = <0x44000000 0x01000000>;
 			no-map;
 		};
+
+		display_reserved: bootsplash@88000000 {
+			reg = <0x88000000 0x05600000>;
+			no-map;
+		};
 	};
 
 	cpus {
-- 
2.25.1

